File: Sopra/finished_project/sopra-fs19-group03-server/src/main/java/ch/uzh/ifi/seal/soprafs19/dto/incoming/GameCardMessage.java
package ch.uzh.ifi.seal.soprafs19.dto.incoming;

import ch.uzh.ifi.seal.soprafs19.constant.GameCard;

public class GameCardMessage implements Message {


    private Long userId;
    private GameCard chosenCard;

    public Long getUserId() {
        return userId;
    }

    public GameCard getChosenCard() {
        return chosenCard;
    }

    public void setUserId(Long userId) {
        this.userId = userId;
    }

    public void setChosenCard(GameCard chosenCard) {
        this.chosenCard = chosenCard;
    }
}

File: Sopra/finished_project/sopra-fs19-group03-client/src/components/shared/models/outgoingMessages/GameCardMessage.js

import "../EnumsJS";


class GameCardMessage {
    constructor(userId, chosenCard){
        this.userId = userId;
        this.chosenCard = chosenCard;
    }
    getUserId(){return this.userId;}
    setUserId(userId){this.userId = userId;}
    getChosenCard(){return this.chosenCard;}
    setChosenCard(chosenCard){this.chosenCard = chosenCard;}
}
export default GameCardMessage;